eox_nelp/edxapp_wrapper/backends/custom_reg_form_nelp.py

```diff
@@ -0,0 +1,16 @@
+"""Backend for custom_reg_form special nelp app.
+
+This file contains all the necessary dependencies from extension of the custom_reg_form of nelp.
+https://github.com/nelc/nelp-custom-registration-fields
+"""
+from custom_reg_form.models import ExtraInfo  # pylint: disable=import-error
```

Comment:
why you didn't add `custom_reg_form` as a normal requirement ?

Reply:
Because is a private  package, (git repo from nelp) =/

Reply:
I think robot edunext has access to that repo

Reply:
> I think robot edunext has access to that repo

To build the image yes, but to run the tests here in github actions and install python requirements?

Reply:
ok, could you set the github action to clone repos with that user ?